src/Uno.UI/Microsoft/UI/Xaml/Controls/ProgressRing/ProgressRing.cs

```diff
@@ -121,28 +129,32 @@ protected override void OnApplyTemplate()
 			UpdateLottieProgress();
 			ChangeVisualState();
 
-			OnForegroundPropertyChanged(this, ForegroundProperty);
-			OnBackgroundPropertyChanged(this, BackgroundProperty);
+			SetLottieForegroundColor();
+			SetLottieBackgroundColor();
```

Comment:
nit: Do we want `// Uno specific:` comments to make it easier when someone tries to sync the control with WinUI repository later?

Reply:
The current implementation is not a import of the original progress ring, as it is relying on a feature we don't fully support (AnimatedVisualPlayer).